index.html

```diff
@@ -26,30 +23,30 @@
       <nav class="desktop-navbar">
         <ul class="desktop-navbar-links">
           <li class="actual-links">
-            <a href=""> Portfolio </a>
+            <a class="nav-links" href="#portfolio"> Portfolio </a>
           </li>
           <li class="actual-links">
-            <a href=""> About </a>
+            <a class="nav-links" href="#about"> About </a>
           </li>
           <li class="actual-links">
-            <a href=""> Contact </a>
+            <a class="nav-links" href="#contact"> Contact </a>
           </li>
           <li>
-            <a href="">
+            <a href="#contact">
               <img
                 class="nav-email"
-                src="./icons/email-icon.svg"
+                src="./images/email-icon.svg"
                 alt="email icon"
               />
             </a>
           </li>
         </ul>
```

Comment:
-  [ ] Good job you implemented the menu :+1:  However there is some issue you need to address with your menu design. :cry:  You are using the wrong  **font-size**  and **font-weight** and the position for your menu option. Please fix it in order to have the similar to the Figma template :smile: 

- [ ] Please adjust your menu option **underlines** to make it **thinner** and use the same **color** from the Figma design, cause it is very important to follow the design from the chosen template in Figma, peer attention to the details is also important  :smile: 

You can check the different screenshots below to check the difference :point_down: 

### Your project
![Screenshot from 2022-07-19 12-09-49](https://user-images.githubusercontent.com/6615394/179752481-854d136c-940b-41fd-9218-6caa975f2153.png)

### Figma template
![Screenshot from 2022-07-19 12-09-32](https://user-images.githubusercontent.com/6615394/179752490-e4c7a2b9-4002-486f-82de-3a40a017fdbe.png)